src/fmeval/transforms/semantic_perturbations.py

```diff
@@ -0,0 +1,301 @@
+import random
+import numpy as np
+from typing import Dict, List, Any
+from abc import abstractmethod
+
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.util import validate_call
+from fmeval.util import require
+
+
+def set_seed(seed: int):
+    random.seed(seed)
+    np.random.seed(seed)
```

Comment:
Question: [`np.random.seed`](https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html) is the legacy method to set the seed, should we switch to using a [generator](https://numpy.org/doc/stable/reference/random/generator.html) instead?

Reply:
Thanks for bringing this to my attention! Will do.